src/main/kotlin/lotto/domain/Statistics.kt

```diff
@@ -0,0 +1,28 @@
+package lotto.domain
+
+import kotlin.math.round
+
+object Statistics {
+
+    var count_3 = 0
+    var count_4 = 0
+    var count_5 = 0
+    var count_6 = 0
+
+    fun countRank(winningCount: Int) {
+        when (winningCount) {
+            3 -> count_3++
+            4 -> count_4++
+            5 -> count_5++
+            6 -> count_6++
+        }
+    }
+
+    fun winningAmount(): Int {
```

Comment:
`Statistics` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private fun winningAmount(): Int {
```